src/AutoRest.CSharp.V3/AutoRest/Plugins/CSharpGen.cs

```diff
@@ -118,7 +118,8 @@ public async Task<bool> Execute(IPluginCommunication autoRest)
                 new Uri(GetRequiredOption(autoRest, "shared-source-folder")).LocalPath,
                 autoRest.GetValue<bool?>("save-inputs").GetAwaiter().GetResult() ?? false,
                 autoRest.GetValue<bool?>("azure-arm").GetAwaiter().GetResult() ?? false,
-                autoRest.GetValue<bool?>("public-clients").GetAwaiter().GetResult() ?? false
+                autoRest.GetValue<bool?>("public-clients").GetAwaiter().GetResult() ?? false,
+                autoRest.GetValue<bool?>("include-unused").GetAwaiter().GetResult() ?? false
```

Comment:
I thought we decided not to add a flag.

Reply:
Yes. After seeing what it does, I'm going to change it to do `internal` by default for any unused models (either no `usage` info or `usage: exception`). However, it uncovered 1 bug testing this (not the deserializer one, a different one), so I'm fixing that.

Reply:
Fixed.